dev/ci/ci-interval.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download interval
+
+( cd "${CI_BUILD_DIR}/interval" && autoconf && ./configure && ./remake "-j${NJOBS}" && ./remake install )
```

Comment:
@silene these kinds of lines are run multiple times, I guess I need a test like
`if [ ! -x ./configure ]; then autoconf && ./condifure; fi`

Reply:
would this one work IYO?

Reply:
That would be a solution.

In theory, `remake` is no different than `make`. That is, it does not recompile anything if the last-modified dates are properly ordered. So, there might be an issue with the way artifacts are handled, e.g., if two slaves have desynchronized clocks. But more probably, the issue just comes from the fact that the `configure` script of Flocq generates some of the `.v` files (due to compatibility issues across various Coq versions), which in turns forces `remake` to recompile their dependencies.